CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs

```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace CandidateApp.ConsoleApp.Models.Candidate
+{
+    class Candidate
+    {
+        public int CID { get; set; }
```

Comment:
What is this CID?

Reply:
Changed as CandidateID

Reply:
Similar to FirstName and LastName you can just say 'Id' as we are in the context of Candidate we need not repeat Candidate in the attributes too.